src/parser.cc

```diff
@@ -134,8 +134,8 @@ int parse( const int argc,
             vm[ExpSwitch::input_path].as<fs::path>(),
             vm[ExpSwitch::output_path].as<fs::path>(),
             devices,
-            vm[CompSwitch::min_sample_i].as<int>(),
-            vm[CompSwitch::max_sample_i].as<int>());
+            vm[CompSwitch::min_sample_i].as<unsigned>(),
+            vm[CompSwitch::max_sample_i].as<unsigned>());
```

Comment:
If I figured it out correctly with gdb, then this line two lines throw an exception:

`terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::bad_any_cast> >'
  what():  boost::bad_any_cast: failed conversion using boost::any_cast
`